php/libraries/NDB_Client.class.inc

```diff
@@ -119,7 +119,9 @@ class NDB_Client
             . $config_additions
         );
         // start php session
-        session_start();
+        if (session_status() == PHP_SESSION_NONE) {
+            session_start();
+        }
```

Comment:
It was causing warnings.

http://php.net/manual/en/function.session-start.php
>4.3.3 	As of PHP 4.3.3, calling session_start() after the session was previously started will result in an error of level E_NOTICE. Also, the second session start will simply be ignored. 

If it's going to "simply be ignored", this pull request doesn't change existing business logic that is not causing failures while removing the E_NOTICE. This Document Repository module causes the warning to happen because it's calling `initialize()`